client/a8c-for-agencies/sections/signup/signup-v2/components/multi-step-form/index.tsx

```diff
@@ -24,9 +31,20 @@ const MultiStepForm = () => {
 			case 1:
 				return <SignupContactForm onContinue={ () => setCurrentStep( 2 ) } />;
 			case 2:
-				return <SignupContactForm onContinue={ () => setCurrentStep( 3 ) } />;
+				return <PersonalizationForm onContinue={ () => setCurrentStep( 3 ) } />;
 			case 3:
-				return <div>Finish</div>;
+				return (
+					<ChoiceBlueprint
+						onContinue={ () => setCurrentStep( 4 ) }
+						onSkip={ () => setCurrentStep( 5 ) }
+					/>
+				);
+			case 4:
+				return <BlueprintForm onContinue={ () => setCurrentStep( 5 ) } />;
+			case 5:
+				return <FinishSignupSurvey onContinue={ () => setCurrentStep( 6 ) } />;
+			case 6:
+				return <h1>Thank you!</h1>;
```

Comment:
Should we translate this?

Reply:
It's a placeholder for now. We will check if we would need this page or not.